WordPress/Classes/ViewRelated/Jetpack Branding/JetpackBannerView.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+
+class JetpackBannerView: UIView {
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    override var intrinsicContentSize: CGSize {
+        return isHidden ? CGSize.zero : super.intrinsicContentSize
+    }
+
+    func setup() {
+
+        translatesAutoresizingMaskIntoConstraints = false
+        backgroundColor = Appearance.jetpackBackgroundColor
+
+        let jetpackButton = makeJetpackButton()
+        addSubview(jetpackButton)
+
+        pinSubviewToAllEdges(jetpackButton)
+        heightAnchor.constraint(equalToConstant: Appearance.jetpackBannerHeight).isActive = true
```

Comment:
We could also remove this constraint here, and add `JEtpackBannerView` as a subview in the container view and apply the constraint externally, or we could expose an `height` property to be overridden: this to allow more flexibility on the view and not just assume that it has to have an height of 44. 
On the other hand, we don't need to do it on this PR, can be safely done in a future one.

Reply:
> this to allow more flexibility on the view and not just assume that it has to have an height of 44.

Is this to allow the banner to expand in height if for example, the user increases text size?

Reply:
> On the other hand, we don't need to do it on this PR, can be safely done in a future one.

Ok, I won't consider this a blocker for this PR and will move ahead with getting it merged. We can continue this discussion here of course.